pmd-cs/src/main/java/net/sourceforge/pmd/cpd/CsTokenizer.java

```diff
@@ -20,26 +20,24 @@ public class CsTokenizer extends AntlrTokenizer {
 
     private boolean ignoreUsings = false;
     private boolean ignoreLiteralSequences = false;
+    private boolean ignoreAttributes = false;
 
     /**
      * Sets the possible options for the C# tokenizer.
      *
      * @param properties the properties
      * @see #IGNORE_USINGS
      * @see #OPTION_IGNORE_LITERAL_SEQUENCES
+     * @see #IGNORE_ANNOTATIONS
      */
     public void setProperties(Properties properties) {
-        ignoreUsings = Boolean.parseBoolean(properties.getProperty(IGNORE_USINGS, Boolean.FALSE.toString()));
-        ignoreLiteralSequences = Boolean.parseBoolean(properties.getProperty(OPTION_IGNORE_LITERAL_SEQUENCES,
-            Boolean.FALSE.toString()));
+        ignoreUsings = getBooleanProperty(properties, IGNORE_USINGS);
+        ignoreLiteralSequences = getBooleanProperty(properties, OPTION_IGNORE_LITERAL_SEQUENCES);
+        ignoreAttributes = getBooleanProperty(properties, IGNORE_ANNOTATIONS);
```

Comment:
Ok, so you are using here the common IGNORE_ANNOTATIONS option. That means, this is also supported at the command line via `--ignore-annotations`.

Could you please also update the doc for this then? Currently it says, only Java supports this option. We would also need to mention in one sentence, that the "annotations" are "attributes" in C#.
Here's the doc page: https://pmd.github.io/latest/pmd_userdocs_cpd.html#cli-options-reference and https://github.com/pmd/pmd/blob/master/docs/pages/pmd/userdocs/cpd/cpd.md